providers/shared/components/datacatalog/id.ftl

```diff
@@ -0,0 +1,266 @@
+                "Children": [
+                    {
+                        "Names": "SerDe",
+                        "Description" : "Sets how the data should be serialised/deserialised",
+                        "Children" : [
+                            {
+                                "Names" : "Library",
+                                "Description": "The library to use for the SerDe process",
+                                "Types" : STRING_TYPE
+                            },
+                            {
+                                "Names" : "Parameters",
+                                "Description" : "Additional key/value parameters to configure the Library - Key of the object is used as key by default",
+                                "SubObjects" : true,
+                                "Children": [
+                                    {
+                                        "Names": "Enabled",
+                                        "Default" : "Include the parameter",
+                                        "Types" : BOOLEAN_TYPE,
+                                        "Default" : true
+                                    },
+                                    {
+                                        "Names": "Key",
+                                        "Description" : "The Key of the parameter",
+                                        "Types" : STRING_TYPE
+                                    },
+                                    {
+                                        "Names": "Value",
+                                        "Description" : "The value of the parameter",
+                                        "Types" : ANY_TYPE
+                                    }
+                                ]
+                            }
+                        ]
+                    },
+                    {
+                        "Names": "Input",
+                        "Description": "The input processer used to proceses the data into table format",
+                        "Types" : STRING_TYPE
+                    },
+                    {
+                        "Names": "Output",
+                        "Description" : "The output processor used to standardise the output of the table",
+                        "Types" : STRING_TYPE
+                    }
+                ]
+            }
+            {
+                "Names": "Crawler",
+                "Description" : "Crawl table to find new columns and update partitioning",
+                "Children" : [
+                    {
+                        "Names" : "Enabled",
+                        "Description": "Enable the use of a Crawler for the table",
+                        "Types" : BOOLEAN_TYPE,
+                        "Default": false
+                    },
+                    {
+                        "Names": "Schedule",
+                        "Description" : "The schedule to run the crawler on",
```

Comment:
CRON format? Maybe add a bit more detail in the description

Reply:
Fixed up, I said that it should be in AWS rate format which should cover it.